### PR TITLE
Update zh_CN.po

### PR DESCRIPTION
**Pull request by tytannial**
Import monster and item translation from Diablo 3 CN Edition and do some random translation fix.

### CODE REVIEW

**Comment by tytannial** [comment]
Diablo 3 stl tool https://github.com/tytannial/D3StlParser

[Full Diablo3 enUS-zhCN Text.zip](https://github.com/diasurgical/devilutionX/files/6968246/Full.Diablo3.enUS-zhCN.Text.zip)

File: Translations/zh_CN.po [hunk 1]
```diff
@@ -1035,110 +1031,95 @@ msgstr "通向"
...
-msgid "Play a demo file"
-msgstr ""
-
-#. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:901
-msgid "Disable all frame limiting during demo playback"
-msgstr ""
-
-#. TRANSLATORS: Commandline Option
-#: Source/diablo.cpp:902
```
Comment (AJenbo) [inline Translations/zh_CN.po:1141]: Looks like you might have reverted to an older version, could you run "Update From Code" and see if that imports the missing bits?

Comment (tytannial) [reply Translations/zh_CN.po:1141]: The pot file not been updated. I can't find those text in
https://github.com/diasurgical/devilutionX/blob/master/Translations/devilutionx.pot
😳 I think the pot file should be updated to 2021-08-09 14:40+0800. The provided devilutionx.pot version is 2021-07-31 17:03+0200.

Comment (AJenbo) [reply Translations/zh_CN.po:1141]: Fair enough, usually I get an option to update directly from the code:
image

File: Translations/zh_CN.po [hunk 2]
```diff
@@ -11,10 +11,6 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: Poedit 3.0\n"
 "Plural-Forms: nplurals=1; plural=0;\n"
-"X-Poedit-SourceCharset: UTF-8\n"
-"X-Poedit-KeywordsList: _;N_\n"
-"X-Poedit-Basepath: ..\n"
-"X-Poedit-SearchPath-0: Source\n"
```
Comment (AJenbo) [inline Translations/zh_CN.po:17]: The removal of thease lines are why you can't update from source

**Comment by AJenbo** [comment]
Merged manually via: https://github.com/diasurgical/devilutionX/commit/c5d48674d11ea5853ec3c9c07687387d8693df72

**Pull request closed without merging**